potimeb/A_Crustacean_Champion.lua

```diff
@@ -1,4 +1,17 @@
+--A_Crustacean_Champion (223136)
+--Phase 2 
+--potimeb
+
 function event_death_complete(e)
-	-- send a signal to the zone_status that I died
-	eq.signal(223097,3); -- NPC: zone_status
+	--signal phase_two_controller
+	eq.signal(223242,1);
+end
+
+function event_signal(e)
+	--signal from A War Shapen Emmisary
+	if e.signal == 1 then
+		e.self:SetSpecialAbility(25, 0);
+		e.self:SetSpecialAbility(24, 0);
+		e.self:SetBodyType(23, true);
```

Comment:
Should use `SpecialAbility.immune_aggro`, `SpecialAbility.immune_aggro_on` and `BT.Monster` here instead (I know we don't do this perfectly everywhere, but we should!)

Reply:
Still missing bodytype